Projects/Algorithmic-Trading/Schemas/Docs-Nodes/T/Trading/Trading-Engine/trading-engine.json

```diff
@@ -22,54 +22,69 @@
                 "language": "EL",
                 "text": "Η ιεραρχία της μηχανής συναλλαγών είναι η δομή δεδομένων που χρησιμοποιείται από το bot συναλλαγών για να διατηρεί τις πληροφορίες χρόνου εκτέλεσης εξαιρετικά προσιτές και διαθέσιμες σε άλλους.",
                 "updated": 1641828391175
+            },
+            {
+                "language": "DE",
+                "text": "Die Trading-Engine-Hierarchie ist die Datenstruktur, die vom Trading-Bot verwendet wird, um Laufzeitinformationen in hohem Maße zugänglich zu halten und anderen zugänglich zu machen.",
+                "updated": 1641762224081
             }
         ]
     },
     "paragraphs": [
         {
             "style": "Block",
             "text": "Content"
-            },
-            {
+        },
+        {
             "style": "Text",
             "text": "You will use this hierarchy for two main purposes:",
             "translations": [
```

Comment:
@harrellbm can you approve the files? Iam honestly a bit unsure about some parts altough they should be fine. Usually at least as far as I know the structure of this parts is something like this:

"paragraphs": [
        {
            "style": "Important",
            "text": "bla bla bla bla bla bla",
            "updated": 1617887115422,
            "translations": [
                {
                    "language": "ES",
                    "text": "bla bla bla bla bla bla bla bla bla ",
                    "updated": 1628732385577
                },

Reply:
The formatting is good to go.  There is actually two "paragraphs". The first is a block format and its text is "Content". A block format is used as a tag so that this section of the docs can be included elsewhere and therefore does not need translated.  The second is your typical paragraph that then has translations attached to it.  Nice eye for spotting something out of the ordinary!